src/NoteMouseHandler/SelectionMouseHandler.js

```diff
@@ -106,10 +105,10 @@ const dragSelectionLeftEdgeAction = (ctx) => (onMouseDown, onMouseMove) => {
   const originalFromTick = ctx.selection.fromTick
   let startPosition
 
-  onMouseDown(e => { startPosition = getLocal(e) })
+  onMouseDown(local => { startPosition = local })
 
-  onMouseMove(e => {
-    const movement = pointSub(getLocal(e), startPosition)
+  onMouseMove(local => {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use 'esversion
*  Too many errors. (59% scanned).


<!-- Comment by [SideCI](https://sideci.com) -->